src/main/java/com/mrsolutions/taskmanangersts/domain/User.java

```diff
@@ -1,81 +1,81 @@
-package com.mrsolutions.taskmanangersts.domain;
-
-import java.io.Serializable;
-
-import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.mapping.Document;
-
-@Document(collection="user")
-public class User implements Serializable {
-	private static final long serialVersionUID = 1L;
-
-	@Id
-	private String id;
-	private String name;
-	private String email;
-	
-	public User() {
-	}
-	
-	public User(String id, String name, String email) {
-		this.id = id;
-		this.name = name;
-		this.email = email;
-	}
-
-	public String getId() {
-		return id;
-	}
-
-	public void setId(String id) {
-		this.id = id;
-	}
-
-	public String getName() {
-		return name;
-	}
-
-	public void setName(String name) {
-		this.name = name;
-	}
-
-	public String getEmail() {
-		return email;
-	}
-
-	public void setEmail(String email) {
-		this.email = email;
-	}
-
-	@Override
-	public int hashCode() {
-		final int prime = 31;
-		int result = 1;
-		result = prime * result + ((id == null) ? 0 : id.hashCode());
-		return result;
-	}
-
-	@Override
-	public boolean equals(Object obj) {
-		if (this == obj)
-			return true;
-		if (obj == null)
-			return false;
-		if (getClass() != obj.getClass())
-			return false;
-		User other = (User) obj;
-		if (id == null) {
-			if (other.id != null)
-				return false;
-		} else if (!id.equals(other.id))
-			return false;
-		return true;
-	}
-
-	@Override
-	public String toString() {
-		return "User [id=" + id + ", name=" + name + ", email=" + email + "]";
-	}
-
-	
-}
+package com.mrsolutions.taskmanangersts.domain;
+
+import java.io.Serializable;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection="user")

```

Comment:
actually for mongo DB we use just a collection for the project or for a set of data.
in this case I guess you can avoid 'collection="user" because mongodb has documents and users, for me, is a document